cmd/fyne/package.go

```diff
@@ -183,11 +183,15 @@ func (p *packager) packageWindows() {
 
 	// write manifest
 	manifest := p.exe + ".manifest"
-	manifestFile, _ := os.Create(manifest)
-	io.WriteString(manifestFile, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n"+
-		"<assembly xmlns=\"urn:schemas-microsoft-com:asm.v1\" manifestVersion=\"1.0\" xmlns:asmv3=\"urn:schemas-microsoft-com:asm.v3\">\n"+
-		"<assemblyIdentity version=\"1.0.0.0\" processorArchitecture=\"*\" name=\""+p.name+"\" type=\"win32\"/>\n"+
-		"</assembly>")
+	manifestGenerated := false
+	if _, err := os.Stat(manifest); os.IsNotExist(err) {
+		manifestGenerated := true
```

Comment:
This should be a "=" not ":=".
The PR test failures are a compilation failure on this line...